libraries/chain/include/scorum/chain/services/pending_bet.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include <scorum/chain/schema/bet_objects.hpp>
+#include <scorum/chain/services/service_base.hpp>
+
+namespace scorum {
+namespace chain {
+
+struct pending_bet_service_i : public base_service_i<pending_bet_object>
+{
+    using pending_bet_call_type = std::function<bool(const base_service_i::object_type&)>;
+
+    virtual void foreach_pending_bets(const game_id_type&, pending_bet_call_type) = 0;
```

Comment:
I would prefer to return a non-owning range to callee instead of passing a functor

Reply:
I am disagree. It is question in productivity